pkgs/applications/version-management/git-and-tools/gitstatus/default.nix

```
@@ -0,0 +1,30 @@
+{callPackage, stdenv, fetchFromGitHub, ...}:
+
+stdenv.mkDerivation rec {
+  pname = "gitstatus";
+  version = "20190506";
```

According to the manual:
```suggestion
  version = "unstable-2019-05-06";
```

---

@nlewo done